src/components/UserProfile/UserProfile.module.css

```diff
@@ -0,0 +1,53 @@
+.user_photo {
+	height: 140px;
+	width: 140px;
+	border-radius: 50%;
+}
+
+.user_profile_container {
+	height: 100%;
+	width: 100%;
+	display: grid;
+	justify-content: center;
+	align-content: flex-start;
+	margin-top: 20px;
+}
+
+.user_profile_content {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	gap: 10px;
+	height: 100%;
+	font-size: var(--fs-h2);
+}
+
+.icon_edit_profile {
+	height: 25px;
+}
+
+.icon_edit_photo {
```

Comment:
Z obecnymi ustawieniami "plusik" ucieka, jak zmienimy rozmiar ekranu:
![image](https://github.com/infoshareacademy/jfdzr11-team-promisesland/assets/123868981/b9b3116b-0b97-4188-8207-4c9ef063ea8d)

Reply:
tak - plusik bedzię uciekał bo jest ustawiony na sztywno za pomocą px, to jest do poprawki zdecydowanie